src/components/events/EventList.tsx

```diff
@@ -102,6 +94,7 @@ export function EventList({
                       />
                     )}
                     <div className={styles.eventCardContent}>
+                      {console.log('event.field_event_tags', event.field_event_tags)}
```

Comment:
This console.log should probably be removed.

Reply:
This is now removed.